ncloud/provider.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
+var NcloudResources map[string]*schema.Resource
+var NcloudDatasources map[string]*schema.Resource
```

Comment:
```suggestion
var NcloudDataSources map[string]*schema.Resource
```
마이너 한데, 카멜케이스 맞추면 좋을 듯 합니다.